packages/lesswrong/components/common/TabNavigationMenu/TabNavigationItem.tsx

```diff
@@ -24,6 +24,9 @@ const styles = (theme: ThemeType): JssStyles => ({
       fontWeight: 600,
     },
   },
+  menuItem: {
```

Comment:
This fixes a bug when long menu items are incorrectly displayed. This is actually an existing bug, but since most of our menu items are short it only affects "Groups & people" and it's only wrong by about 0.5px, which gets rounded up to look correct. The proplem is much more obvious with "Best of the Forum" which is wider.